pkg/porter/list.go

```diff
@@ -122,14 +123,17 @@ func NewDisplayRun(run claims.Run) DisplayRun {
 }
 
 // ListInstallations lists installed bundles.
-func (p *Porter) ListInstallations(opts ListOptions) ([]claims.Installation, error) {
+func (p *Porter) ListInstallations(ctx context.Context, opts ListOptions) ([]claims.Installation, error) {
+	_, log := p.Log.StartSpanNamedByCaller(ctx)
+	defer log.EndSpan()
+
 	installations, err := p.Claims.ListInstallations(opts.GetNamespace(), opts.Name, opts.ParseLabels())
 	return installations, errors.Wrap(err, "could not list installations")
 }
 
 // PrintInstallations prints installed bundles.
-func (p *Porter) PrintInstallations(opts ListOptions) error {
-	installations, err := p.ListInstallations(opts)
+func (p *Porter) PrintInstallations(ctx context.Context, opts ListOptions) (err error) {
```

Comment:
In Porter, I avoid using named return arguments. Since this function isn't using the named return argument err here (it's being immediately redefined below, can you switch it back to just `) error {`?